cmd/apps/vpn-client/vpn-client.go

```diff
@@ -29,6 +29,7 @@ var (
 	localSKStr  = flag.String("sk", "", "Local SecKey")
 	passcode    = flag.String("passcode", "", "Passcode to authenticate connection")
 	killswitch  = flag.Bool("killswitch", false, "If set, the Internet won't be restored during reconnection attempts")
+	dnsAddr     = flag.String("dns", "", "address of DNS want set to tun")
```

Comment:
I will come back and fix the flag text. Not sure what the best wording should be.